<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Experimental Control</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../digitalLock/ui/RepetitionRate.qrc">
    <normaloff>:/openicon/icons/lock-pink.png</normaloff>:/openicon/icons/lock-pink.png</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="CoordinatePlotWidget" name="graphicsLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="actionContinue"/>
   <addaction name="actionSave"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionReload"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>872</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
    <addaction name="actionClear"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionProject"/>
   </widget>
   <widget class="QMenu" name="menuOperation">
    <property name="title">
     <string>Operation</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionPause"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOperation"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetConsole">
   <property name="windowTitle">
    <string>Console Output</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Minimum Level</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="levelComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Maximum Lines</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="linesSpinBox">
         <property name="maximum">
          <number>1000000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="consoleClearButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="Experiment.qrc">
           <normaloff>:/kde/icons/edit-clear.png</normaloff>:/kde/icons/edit-clear.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEditConsole">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="experimentToolBar">
   <property name="windowTitle">
    <string>Experiment Tool Bar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/kde/icons/edit-clear.png</normaloff>:/kde/icons/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/kde/icons/exit.png</normaloff>:/kde/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string> Exit Program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/kde/icons/filesave.png</normaloff>:/kde/icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save Configuration</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/openicon/icons/media-playback-start-7.png</normaloff>:/openicon/icons/media-playback-start-7.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string> Start Experiment</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/openicon/icons/media-playback-pause-7.png</normaloff>
     <normalon>:/kde/icons/edit-clear.png</normalon>:/openicon/icons/media-playback-pause-7.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause Experiment</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/openicon/icons/media-playback-stop-7.png</normaloff>:/openicon/icons/media-playback-stop-7.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop Experiment</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/petersIcons/icons/FPGA.png</normaloff>:/petersIcons/icons/FPGA.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>FPGA Settings</string>
   </property>
  </action>
  <action name="actionContinue">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/kde/icons/player_fwd.png</normaloff>:/kde/icons/player_fwd.png</iconset>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
   <property name="toolTip">
    <string>Continue</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionPulses">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/petersIcons/icons/pulser1.png</normaloff>:/petersIcons/icons/pulser1.png</iconset>
   </property>
   <property name="text">
    <string>Pulses</string>
   </property>
   <property name="toolTip">
    <string>Pulse Program Configuration</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/kde/icons/reload.png</normaloff>:/kde/icons/reload.png</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload Pulse Definition</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDedicatedCounters">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/openicon/icons/gnome-power-statistics.png</normaloff>:/openicon/icons/gnome-power-statistics.png</iconset>
   </property>
   <property name="text">
    <string>DedicatedCounters</string>
   </property>
   <property name="toolTip">
    <string>Dedicated Counters</string>
   </property>
  </action>
  <action name="actionVoltageControl">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/openicon/icons/pictogram-din-w008-electricisty.png</normaloff>:/openicon/icons/pictogram-din-w008-electricisty.png</iconset>
   </property>
   <property name="text">
    <string>VoltageControl</string>
   </property>
   <property name="toolTip">
    <string>Voltage Control</string>
   </property>
  </action>
  <action name="actionProject">
   <property name="text">
    <string>Project</string>
   </property>
   <property name="toolTip">
    <string>Select Project for next program start</string>
   </property>
  </action>
  <action name="actionLogic">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/petersIcons/icons/ChipScope.png</normaloff>:/petersIcons/icons/ChipScope.png</iconset>
   </property>
   <property name="text">
    <string>Logic</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CoordinatePlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>uiModules.CoordinatePlotWidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Experiment.qrc"/>
  <include location="../digitalLock/ui/RepetitionRate.qrc"/>
 </resources>
 <connections/>
</ui>
